tests/test_service.py

```diff
@@ -63,16 +63,36 @@ def setUp(self):
     def tearDown(self):
         db.session.remove()
         db.drop_all()
-    
+
+    def _create_inventory_items(self, count: int):
```

Comment:
I borrowed this from John, it's a pretty useful private method for creating fake `InventoryItems` within our tests and leverages `InventoryItemFactory` under the covers.